coralproject/talk
# Open/Close comments stream

## What does this PR do?

- Adds a status (for open, closed) to the Asset as we not only need to close the status but sometimes open it again. 
- Fix the endpoint and FE to open/close the stream.

## How do I test this PR?

- open the stream
- reload the page
- check that is still open
- close the stream
- reload the page
- check that is still close
